<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TEST_VirtualKeyboard</class>
 <widget class="QMainWindow" name="TEST_VirtualKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TEST_VirtualKeyboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Virtual Keyboard</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_secondaryKeys">
      <property name="text">
       <string>SHow / Hide Secondary Keys</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_input"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit_input"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_input"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_uneditable">
      <property name="currentText">
       <string>Non editable widget</string>
      </property>
      <item>
       <property name="text">
        <string>Non editable widget</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_editable">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>editable widget</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_keyboard"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
